tests/src/Glpi/Tests/CommonDBTestCase.php

```diff
@@ -188,7 +188,8 @@ public function checkInstall($filename = '', $filter = 'glpi_', $when = '') {
                $s_type[0] = str_replace(" COLLATE utf8_unicode_ci", "", $s_type[0]);
                $s_type[0] = str_replace(" CHARACTER SET utf8", "", $s_type[0]);
                if (strpos(trim($s_type[0]), 'text') === 0
-                   || strpos(trim($s_type[0]), 'longtext') === 0) {
+                  || strpos(trim($s_type[0]), 'mediumtext') === 0
```

Comment:
This method is copied from GLPI. It is not available in GLPI 9.2 then I planned to remive it when we will support GLPI 9.3 minimum.

Then we need to apply this fix to GLPI also.